sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -2,187 +2,39 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
   <groupId>com.azure.spring</groupId>
   <artifactId>azure-spring-boot-bom</artifactId>
-  <version>3.14.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot-bom;current} -->
+  <version>3.15.0-beta.1</version>
```

Comment:
will you change the version in aother PR?

Reply:
updated.